common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -284,7 +284,7 @@ public static boolean directBufferPreferred() {
      * Returns the maximum memory reserved for direct buffer allocation.
      */
     public static long maxDirectMemory() {
-        return MAX_DIRECT_MEMORY;
+        return DIRECT_MEMORY_LIMIT;
```

Comment:
It's a little unfortunate that the variable name MAX_DIRECT_MEMORY isn't returned for this method name